live/app/components/course-list.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+import ENV from 'pix-live/config/environment';
+
+const CourseList = Ember.Component.extend({
+
+  // private
+  courses: null,
+  selectedCourse: null,
+
+  didInsertElement () {
+    const that = this;
+    Ember.run.scheduleOnce('afterRender', this, function () {
+      $('button[data-confirm]').click(function () {
+        $('#js-modal-mobile').modal('hide');
+        that.sendAction('startCourse', that.get('selectedCourse'));
+      });
+    });
+
+    if (ENV.environment === 'test') {
+      this.$().on('simulateMobileScreen', function () {
+        that.set('isSimulatedMobileScreen', 'true');
+      });
+    }
+  },
+
+  _isMobile () {
+    if (ENV.environment !== 'test') {
+      return $(window).width() < 767;
+    } else {
+      return this.get('isSimulatedMobileScreen');
+    }
+  },
+
+  actions: {
+    startCourse(course) {
+      if (this._isMobile() && !localStorage.getItem('pix-mobile-warning')) {
```

Comment:
soit mettre le commentaire "display warning only once to user"
soit mettre une sous-fonction "userNotAlreadyWarned"